optimum/bettertransformer/transformation.py

```diff
@@ -79,6 +79,31 @@ def replace_to_bettertransformer(model, config):
     return model
 
 
+def invert_to_old_model(bt_model):
+    r"""
+    Replaces the BT-converted model to its old variant, to be able to safely store the weights
+    of a trained model.
+
+    Args:
+        `model` (`torch.nn.Module`):
+            The input converted model
+    Returns:
+        The invert-converted model
+    """
+
+    for name, module in bt_model.named_children():
+        if len(list(module.children())) > 0:
+            # we may explicitly exclude part of the model to use BetterTransformer
+            invert_to_old_model(module)
+
+        is_invert_compatible = hasattr(module, "old_layer") and module.old_layer is not None
```

Comment:
```suggestion
        can_be_reversed = getattr(module, "orig_layer", None) is not None
```

Reply:
fixed in [39b4a14](https://github.com/huggingface/optimum/pull/676/commits/39b4a14ad6517dcaf91b1854b6a7bb2611d03d91)